### DISCUSSION TITLE
Improve "DNS Setup" page of Domain

### DISCUSSION THREAD

**Discussion by bluepuma77** [postalserver/postal#2641] [Help with using Postal]
### 1. SPF Record

The SPF Record section uses `postal.example.com` even though the domain is known to the page.

`spf.postal.example.com` is mentioned as content, but its unclear if this `spf.postal` itself needs to be created as DNS entry.

### 3. Return Path

In this section the `psrp.postal` prefix and real domain name is used in the text, but not in the code block.

`rp.postal.example.com` is mentioned as value, but its unclear if `rp.postal` itself needs to be created as DNS entry.

### 4. MX Records

"add the following MX records" and "Both records" is stated, but then there is just a single `mx.postal.example.com` in the code block. It's unclear if the entry should be `mx.postal` with type MX and where it should point to.

## To Reproduce

Go to web GUI and access the domains page of email server of organisation.

## Expected behaviour

A clear and concise description of what the user is expected to do.

## Screenshots
<img width="715" alt="Postal-Domain-Setup" src="https://github.com/postalserver/postal/assets/5221187/da24640a-7dd1-443e-b88e-566a4d2b3e3e">

## Additional information/context

This is how my DNS provider settings look like, I need the key/subdomain, a type and a value.
<img width="912" alt="DNS-Domain-Setup" src="https://github.com/postalserver/postal/assets/5221187/38347ad1-b81e-4c05-bd53-35fd02f66d98">

**Comment by willpower232** [comment]
this means you either haven't changed the example.com's in the config file or you haven't restarted Postal after doing so

**Reply by bluepuma77** [reply]
Sorry, I think I missed to read through all of [DNS config](https://docs.postalserver.io/install/dns-configuration), that's why I didn't understand where `spf.` and `rp.` are coming from.
